courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/3_keras_sequential_api.ipynb

```diff
@@ -156,6 +156,13 @@
     "We use a python dictionary comprehension to create the feature columns for our model, which is just an elegant alternative to a for loop."
```

Comment:
<p>What about that instead:</p><p> </p><p>Create a feature column dictionary that we will use when building our deep neural network below. The keys should be the element of the&nbsp;<code>INPUT_COLS</code>&nbsp;list, while the values should be numeric feature columns. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/704/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>yeah. Sounds better. I'll change that. thanks!</p>